packages/grafana-ui/src/components/ColorPicker/ColorPickerPopover.tsx

```diff
@@ -113,10 +116,15 @@ export class ColorPickerPopover<T extends CustomPickersDescriptor> extends React
 
   render() {
     const { theme } = this.props;
-    const colorPickerTheme = theme.type || GrafanaThemeType.Dark;
+    const styles = getStyles(theme);
     return (
-      <div className={`ColorPickerPopover ColorPickerPopover--${colorPickerTheme}`}>
-        <div className="ColorPickerPopover__tabs">
+      <div
+        className={cx(
+          styles.colorPickerPopover,
+          theme.type === 'light' ? styles.colorPickerPopoverLight : styles.colorPickerPopoverDark
```

Comment:
```suggestion
          theme.type === GrafanaThemeType.Light ? styles.colorPickerPopoverLight : styles.colorPickerPopoverDark
```

It's a good practice to use defined types rather than static values, as long term if the type definition changes, you don't have to update all static values used :)